connectors/kafka.mdx

```diff
@@ -36,6 +36,8 @@ A Kafka connection has several required and optional fields:
 | topic | The name of the Kafka topic to read from or write to | Yes |
 | type | The type of table (either 'source' or 'sink') | Yes | `source` |
 | source.offset | The offset to start reading from (either 'earliest' or 'latest') | If source | `earliest` |
+| source.read_mode | The read mode to use (either 'read_committed' or 'read_uncommitted') | If source | `read_committed` |
```

Comment:
I think required is No for both of these

Reply:
Also I think this could use more explanation of what it means to set this to `read_committed`